corehq/apps/settings/forms.py

```diff
@@ -140,9 +139,9 @@ def clean_token(self):
 
 class HQTwoFactorMethodForm(MethodForm):
 
-    def __init__(self, **kwargs):
-        super(HQTwoFactorMethodForm, self).__init__(**kwargs)
-        if not settings.ALLOW_PHONE_AS_DEFAULT_TWO_FACTOR_DEVICE:
+    def __init__(self, *, allow_phone, **kwargs):
```

Comment:
I might change this variable to be called `allow_phone_2fa` or `allow_phone_as_two_factor_device` so there's less doubt for someone reading the code.